File: Suhyun/Level1/1월 1주차/수박수/Subac.java
public class Subac {
    public static void main(String[] args) {
        int n=3;
        String answer="";
        
        //6.13
        for(int i=0; i<n; i++){
            answer+= i%2==0 ? "수" : "박";
        }

        // 6.84
        // int count=1;
        // for(int i=0; i<n; i++){
        //     if(count%2!=0){
        //         answer=answer+"¼ö";
        //     }
        //     else{
        //         answer=answer+"¹Ú";
        //     }
        //     count++;
        // }

        System.out.print(answer);
        //return asnwer;


    }
}

File: Suhyun/Level1/1월 1주차/수박수/Subac.js
function solution(n) {
    //0.28s
    let n=3;
    let answer="";

    for(let i=0; i<n; i++){
        answer+= i%2==0 ? '수' : '박';
    }
}
console.log(answer);

//0.05s
function solution(n) {
    return '¼ö¹Ú'.repeat(n/2) + (n%2 === 1 ? '¼ö' : '');
   
}
